lib/links.js

```diff
@@ -10,19 +10,20 @@ const { JSDOM } = require('jsdom');
  * Most of this class is ported from node-review
  */
 class LinkParser {
-  constructor(repo, html) {
+  constructor(owner, repo, html) {
+    this.owner = owner;
     this.repo = repo;
     this.OP = JSDOM.fragment(html);
   }
 
   getFixesUrlsFromArray(arr) {
-    const repo = this.repo;
+    const { owner, repo } = this;
     const result = [];
     for (const item of arr) {
       const m = item.match(FIX_RE);
       if (!m) continue;
       const fix = m[3];
-      const url = fix.replace(/^#/, `${repo}#`).replace('#', '/issues/');
+      const url = fix.replace(/^#/, `${owner}/${repo}#`).replace('#', '/issues/');
```

Comment:
Wait..can we use `getRefUrlFromOP()` for this? That one looks more reliable actually.

Reply:
Also this will skip a cross-repository fix, that one seems to solve it as well.